configure

```diff
@@ -2150,6 +2155,44 @@ int main() {
     }
 }
 
+check_higgstools_incl() {
+    check_header "Higgs/Predictions.hpp" "$higgstools_inc_dir"
+    if test "x$found_hdr" = "x" ; then
+        if test "x$enable_higgstools" = "xyes" ; then
+            message "Error: HiggsTools's Predictions.hpp must be installed if you want"
+            message "   to use HiggsTools, see https://gitlab.com/higgsbounds/higgstools"
+            message "   You can use the --with-higgstools-incdir= option to specify the"
+            message "   location of Higgs/Predictions.hpp and the other headers."
+            exit 1
+        else
+            return 1
+        fi
+    else
+      HIGGSTOOLSFLAGS="-Imodel_specific/SM -I$higgstools_inc_dir"
```

Comment:
Not sure about those lines. They are mostly a copy from the analogues GM2calc functions. I've added `-Imodel_specific/SM` because HiggsTools interface will require a built-in SM. Also, HiggsTools has no central header file, so I've went with checking for `Predictions.hpp`.

Reply:
I think it would be best to not add `-Imodel_specific/SM` here, but instead modify `get_model_specific_corrections()` to check if HiggsTools is enabled. If it is, one would add the model-specific `SM` to the `ho` variable. The effect would be that the required `-Imodel_specific/SM` would be added automatically.

Reply:
`get_model_specific_corrections` is chain called through `update_model_makefiles`. But `update_model_makefiles` is called before `enable_higgstools`. Isn't variable `enable_higgstools` undefined at this point (at best it's set to `automatic`). It still has to be determined if it's `true` or `false`.

Reply:
I'm confused. Currently we have
```
      # always build SM, because it is needed for the effective couplings module
      local ho="SM"
```
Does this mean that we are adding `model_specific/SM` to includes anyway?